### DISCUSSION TITLE
Orthographic View with MVT Tile Layer

### DISCUSSION THREAD

**Discussion by cornhundred** [visgl/deck.gl#5556] [Q&A] [answered]
Does anyone know if it is possible to use the orthographic view with MVT tiles? Is there a tool that can generate MVT tiles in a simple projection to visualize 2 dimensional data? It does not seem that tippecanoe supports this https://github.com/mapbox/tippecanoe/issues/907. I'm not sure if https://github.com/tilezen/mapbox-vector-tile/issues/109 supports this.

We have this example using some MVT tiles hosted on GitHub:
https://observablehq.com/@cornhundred/deck-gl-mvt-layer-custom

but I would like to try using MVT tiles to visualize non-geographic data (e.g. biological data) using the Deck.gl MVT layer

**Comment by Pessimistress** [comment, marked as answer]
You need to use the TileLayer. The MVT spec is geospatial only.

**Reply by cornhundred** [reply]
Thanks, @Pessimistress. I wonder if it would be possible to hack our data into lat/lng geospatial format by performing the inverse mercator projection on the data such that it renders as a grid in the geographic lat/lng coordinates.
